PatientRegistrationApp/src/components/GreetingPage.vue

```diff
@@ -75,22 +78,57 @@
 	</v-container>
 </template>
 
+
+
 <script>
+import EventBus from '../eventBus'
 
-  export default {
-    name: 'GreetingPage',
+export default {
+	name: "GreetingPage",
 	data: () => ({
-	items: ['English','Spanish'],
+		preferredLanguageOptions: ['English','Spanish'],
+		preferredLanguage: ''
+	
 	}),
+      
+     
+ 
 	methods: {
-		singleRegistration()
+  
+	singleRegistration()
 		{
 			this.$emit("singleRegistration");
 		},
 		householdRegistration()
 		{
 			this.$emit("householdRegistration");
+		},
+	sendGreetingPageInfoToReviewPage()
+	{
+		const greetingPagePayload = {
+		preferredLanguage: this.preferredLanguage,
 		}
-	}
-  }
+		EventBus.$emit('DATA_LANGUAGE_INFO_PUBLISHED', greetingPagePayload)
+	}, 
+	verifyFormContents()
+    {
+		//add logic to check form contents
+		var valid = true
+		var message = "Woops! You need to enter the following fields:"
+	
+			if (this.preferredLanguage == "") {
+			message += "*Preferred Language"
```

Comment:
It'd be best to get rid of the "*" and just keep it to the field name in human readable text without any special characters. Also, why is the notification plural? It shouldn't be.

![image](https://user-images.githubusercontent.com/67079030/97830310-21096980-1c92-11eb-9ad9-b9dcd20c1ebb.png)